README.md

```diff
@@ -22,16 +22,34 @@ kpack also provides a build type to execute a single Cloud Native Buildpack imag
 ### Creating an Image Resource
 
 1. Create a builder resource. This resource tracks a builder on registry and will rebuild images when the builder has updated buildpacks. 
-    ```yaml
-    apiVersion: build.pivotal.io/v1alpha1
-    kind: Builder
-    metadata:
-      name: sample-builder
-    spec:
-      image: cloudfoundry/cnb:bionic
-      imagePullSecrets: # optional, if not set builder must be public
-      - name: builder-secret
-    ```
+    Two types of builders can be created, Cluster Scoped Builders and Namespace Scoped Builders
+
+    1. Namespace scoped builders
+        ```yaml
+        apiVersion: build.pivotal.io/v1alpha1
+        kind: Builder
+        metadata:
+          name: sample-builder
+        spec:
+          image: cloudfoundry/cnb:bionic
+          # imagePullSecrets: # Use these secrets if credentials are needed to pull the builder
+          # - name: builder-secret
+        ```
+
+   1. Cluster Scoped Builders
+        ```yaml
+        apiVersion: build.pivotal.io/v1alpha1
+        kind: ClusterBuilder
+        metadata:
+          name: cluster-sample-builder
+        spec:
+          image: cloudfoundry/cnb:bionic
+          # imagePullSecrets: # Use these secrets if credentials are needed to pull the builder
```

Comment:
There is some discussion of [ClusterSecrets](https://github.com/kubernetes/kubernetes/issues/70147) and some cluster scoped types can reference namespace scoped types like ClusterRoleBindings. However, if the secret was in another namespace from the namespace a build was used in, it would need to copy the secret to the build's namespace.

For now, it probably makes sense to only support public builders and remove the imagePullSecrets.

Reply:
Removed the pull secrets from the cluster builder